packages/components/app/styles/components/copy/snippet.scss

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// COPY > SNIPPET
```

Comment:
@andgen404 can you especially check the tokens to make sure I got them correct? Thank you!